pkg/runtime/runtime.go

```diff
@@ -2633,26 +2685,29 @@ func (a *DaprRuntime) processComponentAndDependents(comp componentsV1alpha1.Comp
 	return nil
 }
 
-func (a *DaprRuntime) doProcessOneComponent(category components.Category, comp componentsV1alpha1.Component) error {
+// Attempts to create and initialize a single component of a specific type.
+//
+// Returns whether the component was found and initialized, or any error that occurred.
+func (a *DaprRuntime) doProcessOneComponent(category components.Category, comp componentsV1alpha1.Component) (bool, error) {
 	switch category {
 	case components.CategoryBindings:
 		return a.initBinding(comp)
 	case components.CategoryPubSub:
 		return a.initPubSub(comp)
 	case components.CategorySecretStore:
-		return a.initSecretStore(comp)
+		return true, a.initSecretStore(comp)
```

Comment:
Can you change the signature of these methods to return a boolean too? For now they can always return `true` until we support them as pluggable